dataflow/src/main/java/org/checkerframework/dataflow/expression/JavaExpression.java

```diff
@@ -799,6 +800,59 @@ public JavaExpression atFieldAccess(JavaExpression receiver) {
     return ViewpointAdaptJavaExpression.viewpointAdapt(this, receiver);
   }
 
+  ///
+  /// Operations on Method Calls and Methods
+  ///
+
+  /**
+   * Returns a list of {@code MethodCall} subexpressions from a {@code MethodCall} expression.
```

Comment:
This seems to work only for chained/"fluent" method calls.  That should be documented, but I'm not sure why it is necessary.